org.freedesktop.Sdk.Extension.haskell.json

```diff
@@ -0,0 +1,96 @@
+{
+    "id": "org.freedesktop.Sdk.Extension.haskell",
+    "branch": "19.08",
+    "runtime": "org.freedesktop.Sdk",
+    "runtime-version": "19.08",
+    "build-extension": true,
+    "sdk": "org.freedesktop.Sdk",
+    "separate-locales": false,
+    "appstream-compose": false,
+    "build-options": {
+        "prefix": "/usr/lib/sdk/haskell"
+    },
+    "modules": [
+        {
+            "name": "ghc",
+            "buildsystem": "autotools",
+            "make-args": [
+                "install"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://downloads.haskell.org/~ghc/latest/ghc-8.10.1-x86_64-fedora27-linux.tar.xz",
```

Comment:
Should use flatpak-external-data-checker for detecting when updates are available.

Reply:
I'll try to add that later today for all the links.